tools/lsp/wasm_main.rs

```diff
@@ -203,15 +226,29 @@ impl SlintServer {
         wasm_bindgen_futures::future_to_promise(async move {
             let _lock = ReentryGuard::lock(guard).await;
             let uri: lsp_types::Url = serde_wasm_bindgen::from_value(uri)?;
+            let mut progress = ctx
+                .server_notifier
+                .progress_reporter(
+                    None,
+                    "Opening document".into(),
+                    Some(format!("opening: {}@{}", uri.clone(), version)),
```

Comment:
It seems the first letter is capital in the other messages. Intentional lower case here?